### ISSUE TITLE
Attachments do not upload, a file appears but of size 0k

### ISSUE THREAD

**Issue by GoogleCodeExporter** [willmyers428/google-email-uploader-mac#11]
```
What steps will reproduce the problem?
1.Upload any email with an attachment
2.Look at email in Gmail, file is there but of 0k won't open etc..

What is the expected output? What do you see instead?


Apple Mail 3.6 on Leopard


Just uploaded 10000 emails, no attachments uploaded properly


```

Original issue reported on code.google.com by `cmcnau...@gmail.com` on 6 Feb 2010 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
If your message files for Apple Mail contain separate sections for the text and 
the
enclosures, similar to this, then the enclosures should upload.


--Apple-Mail-2--408920290
Content-Type: text/plain;
    charset=US-ASCII;
    format=flowed
Content-Transfer-Encoding: 7bit

some text here

--Apple-Mail-2--408920290
Content-Disposition: inline;
    filename=pic_01.jpg
Content-Type: image/jpeg;
    x-unix-mode=0644;
    x-mac-type=4A504547;
    name="pic_01.jpg"
Content-Transfer-Encoding: base64

/9j/4RbORXhpZgAATU0AKgAAAAgACQESAAMAAAABAAEAAAESAAMAAAABAAEAAAEaAAUAAAABAAAA
egEbAAUAAAABAAAAggEoAAMAAAABAAIAAAExAAIAAAAQAAAAigEyAAIAAAAUAAAAmgE8AAIAAAAQ
AAAArgITAAMAAAABAAEAAAAAAL4ASAAAAAEAAABIAAAAAQAAUXVpY2tUaW1lIDcuNi4yADIwMDk6
Djxxq/hv4oWdlYSuTbW66dc+ZJujvba6jZTG0eMKFYFwR3NeD+IviRr2meMNZ8OXEr3Nhb35sIxJ
KwCxCTfHgdiuQuf4gvNfXUZyqUlOW/Nb5WifkGKqSVNNdz//2Q==

```

Original comment by `gregrobbins` on 8 Feb 2010 at 8:39

**Comment by GoogleCodeExporter** [comment]
```
Here is an example of raw text from Apple Mail which resulted in 2 0k 
attachments.
The attachments were fine in a message received in one of my gmail accounts.  
They arrived in my Apple Mail 
program via IMAP with attachments intact. When I used the uploader to transfer 
to my Business account, the 
attachments wer recognized but no content.

I have since used the uploader successfully with attachments from Apple Mail 
where the message with 
attachment was directly received via POP 
Colin

Delivered-To: cmcnaught@gmail.com
Received: by 10.216.10.206 with SMTP id 56cs30272wev; Sun, 24 Jan 2010
 10:17:53 -0800 (PST)
Received: by 10.90.11.7 with SMTP id 7mr4816839agk.79.1264357071778; Sun, 24
 Jan 2010 10:17:51 -0800 (PST)
Return-Path: <crow@gwmail.gwu.edu>
Received: from mail-yx0-f200.google.com (mail-yx0-f200.google.com
 [209.85.210.200]) by mx.google.com with ESMTP id
 23si9095862gxk.3.2010.01.24.10.17.51; Sun, 24 Jan 2010 10:17:51 -0800 (PST)
Received-SPF: neutral (google.com: 209.85.210.200 is neither permitted nor
 denied by best guess record for domain of crow@gwmail.gwu.edu)
 client-ip=209.85.210.200;
Authentication-Results: mx.google.com; spf=neutral (google.com: 209.85.210.200
 is neither permitted nor denied by best guess record for domain of
 crow@gwmail.gwu.edu) smtp.mail=crow@gwmail.gwu.edu
Received: by mail-yx0-f200.google.com with SMTP id 38so112230yxe.6 for
 <cmcnaught@gmail.com>; Sun, 24 Jan 2010 10:17:51 -0800 (PST)
MIME-Version: 1.0
Sender: crow@gwmail.gwu.edu
Received: by 10.150.235.19 with SMTP id i19mr7245657ybh.92.1264357070723; Sun,
 24 Jan 2010 10:17:50 -0800 (PST)
Date: Sun, 24 Jan 2010 13:17:50 -0500
X-Google-Sender-Auth: 06b8a0de62467feb
Message-ID: <5147c04f1001241017i472c1c33t42b5d5e43c50f1a@mail.gmail.com>
Subject: Djesben draft marketing materials
From: Christian Crowley <crow@gwu.edu>
To: Colin McNaught <cmcnaught@gmail.com>
Content-Type: multipart/mixed; boundary=000e0cd25a9011ddec047ded1245


--000e0cd25a9011ddec047ded1245
Content-Type: multipart/alternative;
    boundary=000e0cd25a9011ddd7047ded1243


--000e0cd25a9011ddd7047ded1243
Content-Type: text/plain;
    charset=ISO-8859-1
Content-Transfer-Encoding: 7bit

Greetings,

.....text of message......

--000e0cd25a9011ddd7047ded1243
Content-Type: text/html;
    charset=ISO-8859-1
Content-Transfer-Encoding: quoted-printable

.....Text of message in html.....

--000e0cd25a9011ddd7047ded1243--

--000e0cd25a9011ddec047ded1245
Content-Disposition: attachment;
    filename="Djesben business card 01-21-2010.doc"
Content-Type: application/msword;
    name="Djesben business card 01-21-2010.doc"
Content-Transfer-Encoding: base64

0M8R4KGxGuEAAAAAAAAAAAAAAAAAAAAAPgADAP7/CQAGAAAAAAAAAAAAAAADAAAAAAEAAAAAAAAA
EAAAAgEAAAEAAAD+////AAAAAP0AAAD+AAAA/wAAAP//////////////////////////////////
////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////
////////////////////////////////////////////////////////////////////////////
///////////////////////////////////////////////////////////////////////////s
pcEAA4AJBAAA+BK/AAAAAAAAEAAAAAAACAAA0AkAAA4AYmpiamNUY1QAAAAAAAAAAAAAAAAAAAAA
AAAJBBYAdhIAAAE+AQABPgEACgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA3QAAAAAAAAD//w8AAAAA
AAAAAAD//w8AAAAAAAAAAAD//w8AAAAAAAAAAAAAAAAAAAAAALcAAAAAALYFAAAAAAAAtgUAAPkS

and so on


--000e0cd25a9011ddec047ded1245--
```

Original comment by `cmcnau...@gmail.com` on 9 Feb 2010 at 11:12

**Comment by GoogleCodeExporter** [comment]
```


Missed the break between the two Word documents which contained the following:

.........
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
--000e0cd25a9011ddec047ded1245
Content-Disposition: attachment;
    filename="Djesben info-card 01-21-2010.doc"
Content-Type: application/msword;
    name="Djesben info-card 01-21-2010.doc"
Content-Transfer-Encoding: base64

0M8R4KGxGuEAAAAAAAAAAA..............

```

Original comment by `cmcnau...@gmail.com` on 9 Feb 2010 at 11:19

**Comment by GoogleCodeExporter** [comment]
```
I've finally reproduced a case where uploading from a Mail.app IMAP archive 
creates
an enclosure of 0K. In particular, I had a message with three enclosures (jpg, 
txt,
pdf), and the data for the pdf was stored not in the message's emlx file but 
rather
in a separate emlxpart file beside the emlx file.

The uploader does not currently merge emlxpart files into the emlx files before
uploading, but apparently should.
```

Original comment by `gregrobbins` on 14 Feb 2010 at 11:37
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in release 1.1.1.
```

Original comment by `gregrobbins` on 24 Feb 2010 at 8:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Its not fixed. Ver 1.1.2
```

Original comment by `alfred.w...@silans.com` on 28 Feb 2011 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
I am experiencing the same issue.

Is there a way to prepare email/attachments locally so that it will upload 
successfully?
```

Original comment by `roseabil...@gmail.com` on 15 Mar 2011 at 11:41

**Comment by GoogleCodeExporter** [comment]
```
It is still not fixed in ver 1.1.2

snow leopard 10.6.8

i tried with apple mail, thunderbird for mac and outlook (which is not 
supported)

i planned to move my mails to gmail business account. 


is there any solution to fix this problem??

thanks for any advice


markus
```

Original comment by `k...@kusi.li` on 12 Jul 2011 at 9:12

**Comment by GoogleCodeExporter** [comment]
```
ver 1.2 still seems to have this issue... just uploaded 20,000 mails, 
apparently with NO attachemnts coming thorough, all display as "0K"...

has this issue been looked into or is there an update?

thanks...
a

```

Original comment by `an...@antonkusters.com` on 26 Jan 2012 at 9:26
